ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2771,5 +2771,155 @@
     "name": "type-blank.svg",
     "mediaType": "image/svg+xml"
   }
- }
+ },
+  {
+    "description": "Ceylon stack with Java, Javascript and Dart backends on CentOS.",
+    "creator": "David Festal",
+    "scope": "advanced",
+    "name": "Ceylon with Java Javascript Dart on CentOS",
+    "id": "ceylon-java-javascript-dart-centos",
+    "source": {
+      "origin": "eclipse/centos_ceylon_nodejs_dart",
+      "type": "image"
+    },
+    "components": [
+      {
+        "version": "---",
+        "name": "CentOS"
+      },
+      {
+        "version": "1.8.0_45",
+        "name": "JDK"
+      },
+      {
+        "version": "3.2.2",
```

Comment:
based on https://github.com/eclipse/che-dockerfiles/blob/master/recipes/centos_jdk8/Dockerfile it seems it's 3.3.9